apps/web/CLOUDFLARE_DEPLOYMENT.md

```
@@ -0,0 +1,392 @@
+3. Add all production env vars
+4. Settings → Functions
+5. Set compatibility date to `2025-05-05`
+6. Enable `nodejs_compat` compatibility flag
+
+## Step 5: Set Up Tasks Service on Cloudflare Containers
+
+The tasks service requires FFmpeg, so it must run on Cloudflare Containers (not standard Workers).
+
+### Create Dockerfile for Tasks Service
+
+See `apps/tasks/Dockerfile.cloudflare` (created by setup script).
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)